packages/theme/pages/MyAccount.vue

```diff
@@ -139,19 +89,38 @@ export default {
     margin: 0 auto;
   }
 }
+.breadcrumbs {
+  margin: var(--spacer-base) 0 var(--spacer-lg);
+}
 .my-account {
-  @include for-mobile {
-    --content-pages-sidebar-category-title-font-weight: var(
-      --font-weight--normal
-    );
-    --content-pages-sidebar-category-title-margin: var(--spacer-sm)
-      var(--spacer-sm) var(--spacer-sm) var(--spacer-base);
+  display: flex;
+
+  &__sidebar {
+    width: 250px;
+    font-family: var(--font-family--secondary);
+
+    h1 {
+      font-weight: var(--content-pages-sidebar-title-font-weight);
+      font-size: var(--h3-font-size);
+      margin: var(--content-pages-sidebar-title-margin, 0 0 var(--spacer-xl) 0);
+    }
+    h2 {
+      font-weight: var(--font-weight--bold);
+      font-size: var(--font-size--lg);
+      margin: var(--content-pages-sidebar-category-title-margin, var(--spacer-sm) 0);
+    }
+
+    a {
+      display: block;
+      color: var(--c-dark-variant);
+      margin: var(--spacer-base) 0;
+    }
+    .nuxt-link-active {
+      color: var(--c-primary)
+    }
   }
-  @include for-desktop {
-    --content-pages-sidebar-category-title-margin: var(--spacer-xl) 0 0 0;
+  &__view {
+    width: calc(100% - 300px);
```

Comment:
What's the fixed 300px for? If it's related to the sidebar and reused somewhere (perhaps, line 99?), I'd consider creating a meaningful variable + calculation, to avoid magic numbers.